source/codegen/templates/service_helpers.mako

```diff
@@ -57,7 +57,9 @@ ${initialize_input_params(function_name, non_ivi_params)}\
 ${initialize_output_params(output_parameters)}\
 </%block>\
         status = library_->${function_name}(${service_helpers.create_args(parameters)});
-        if (status == kErrorReadBufferTooSmall || status > ${common_helpers.camel_to_snake(size_param['cppName'])}) {
+        // We cast status into ${common_helpers.camel_to_snake(size_param['cppName'])} above, so it's safe to cast
```

Comment:
I'd vote to change this to a mako comment, but it's OK like this.

Reply:
Yeah, I went back and forth on this. I'll change it.